src/logging.js

```diff
@@ -147,9 +146,11 @@ export function makeRequestMiddleware(logger: Logger): Promise<Middleware> {
               }),
           )
         : /**
-           * Otherwise, we're using the Google middleware
+           * Otherwise, we're using the Google middleware.
+           * This does some nice things including adding the appropriate
+           * logging transport for us.
            */
-          lw.express.makeMiddleware(logger);
+          lw.express.makeMiddleware(logger, getTransport(false));
```

Comment:
Ah, ok, I see it now!

Reply:
> OK! I don't fully know the makeMiddleware syntax but this seems plausible.
> 
> But I don't understand why your code is so different from graphql-gateway. Of course, maybe graphql-gateway isn't working quite right, but I feel like it is. See what we do in services/graphql-gateway/src/main.js. Maybe you want to try the same thing here.

I think, based on looking at the graphql-gateway code, you actually do have this issue. You may just not have noticed the duplication.